tests/mocha/workspace_test.js

```diff
@@ -86,7 +88,7 @@ suite('Workspace', function() {
       createVarBlocksNoEvents(this.workspace, ['id1', 'id1', 'id2']);
     });
 
-    test('deleteVariableInternal_', function() {
+    test('deleteVariableInternal_(\'id1\')', function() {
```

Comment:
I would maybe just have it be 'deleteVariableInternal_(id1)' gets the point across without having it look cluttered

Reply:
Done